pkg/driver/microphone/microphone.go

```diff
@@ -153,6 +149,11 @@ func (m *microphone) AudioRecord(inputProp prop.Media) (audio.Reader, error) {
 		closeDeviceOnce.Do(func() {
 			cancel() // Unblock onRecvChunk
 			device.Uninit()
+
+			if m.chunkChan != nil {
+				close(m.chunkChan)
+				m.chunkChan = nil
+			}
```

Comment:
As it's in Once,
```suggestion
			close(m.chunkChan)
			m.chunkChan = nil
```

Reply:
I left that there because we don't check if we've called Open, so the channel may be nil, hypothetically